lib/masdb/schema.ex

```diff
@@ -15,6 +15,14 @@ defmodule Masdb.Schema do
   @enforce_keys [:name, :replication_factor]
   defstruct [:name, :replication_factor, columns: [], creation_time: Masdb.Timestamp.get_timestamp()]
 
+  def get_pk(%Masdb.Schema{columns: cols}) do
```

Comment:
No, filter_map will return every pk we have. Would you like this function to be renamed get_pks ?